src/gridjs-vue.vue

```diff
@@ -159,6 +159,11 @@ export default {
     // instantiate grid.js
     if (this.wrapper && (this.options.data || this.options.from || this.options.server)) {
       this.grid = new Grid(this.options).render(this.wrapper)
+	  this.grid.on('rowClick', (...args) => {this.$emit('grid-row-click', args)})
```

Comment:
do we have to emit these events? Also, why are we renaming the events to `grid-row-click` instead? This can be a bit confusing for user.